src/cli/run.js

```diff
@@ -68,7 +68,14 @@ const inputFiles = cliFiles.length
 
 /** Step (2) */
 
-const files = processFiles(inputFiles, (f) => readFileSync(f, 'utf8'));
+const files = processFiles(inputFiles, (f) => {
+    if (!existsSync(f)) {
+        if (existsSync(f + '.js')) {
+            return readFileSync(f + '.js', 'utf8');
+        }
+    }
+    return readFileSync(f, 'utf8');
```

Comment:
I think this would be easier to read if the conditionals were flattened out:

```js
if (existsSync(f) {
  return readFileSync(f, 'utf8');
}
if (existsSync(f + '.js')) {
  return readFileSync(f + '.js', 'utf8');
}
```